packages/contracts/contracts/PassportFactory.sol

```diff
@@ -17,9 +17,11 @@ contract PassportFactory is Ownable {
     uint id
   );
 
-  function create() public {
+  function create(string memory name_, string memory symbol_, uint256[] memory tokenIds, uint256 price) public {
+    require(tokenIds.length <= 100, "Could only mint max 100 passports");
```

Comment:
setting a max for now, but this might not even be necessary, since I think the gas cost will scale with membership supply